core/integration/features/verified/closeout-2 partyies_lognormal.feature

```diff
@@ -0,0 +1,130 @@
+    When the parties place the following orders:
+      | party     | market id | side | volume| price | resulting trades | type       | tif     | reference   |
+      | trader2   | ETH/DEC19 | buy  | 40    | 50    | 0                | TYPE_LIMIT | TIF_GTC | buy-order-3 |
+    Then the order book should have the following volumes for market "ETH/DEC19":
+      | side | price  | volume   |
+      | buy  | 5      | 5        |
+      | buy  | 50     | 4040     | 
+      | sell | 1000   | 10       | 
+      | sell | 1055   | 223      | 
+
+    And the parties should have the following margin levels:
+      | party     | market id  | maintenance | search | initial | release |
+      | trader2   | ETH/DEC19  | 321         | 481    | 642     | 963     |
+      # margin level = OrderSize*MarkPrice*RF = 40*10*0.801225765=321
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin   | general |
+      | trader2 | USD   | ETH/DEC19 | 642      | 1358    |
+
+    # setup trader3 position and close it out
+    When the parties place the following orders:
+      | party      | market id | side | volume| price | resulting trades | type       | tif     | reference      |
+      | trader3    | ETH/DEC19 | buy  | 10    | 100   | 0                | TYPE_LIMIT | TIF_GTC | buy-position-31 |
+      
+    Then the order book should have the following volumes for market "ETH/DEC19":
+      | side | price  | volume   |
+      | buy  | 5      | 5        |
+      | buy  | 45     | 278966   |
+      | buy  | 50     | 40       | 
+      | buy  | 100    | 10       | 
+      | sell | 1000   | 10       | 
+      | sell | 1055   | 223      | 
+
+    And the parties should have the following margin levels:
+      | party     | market id  | maintenance | search | initial | release |
+      | trader3   | ETH/DEC19  | 81          | 121    | 162     | 243     |
+
+    # how come trader3's order is not rejected? they only have 90 in the collateral and the initial margin level is 162
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin   | general |
+      | trader2 | USD   | ETH/DEC19 | 642      | 1358    |
+      | trader3 | USD   | ETH/DEC19 | 90       | 0       |
+
+    And the insurance pool balance should be "0" for the market "ETH/DEC19"
+
+    Then the parties should have the following profit and loss:
+      | party      | volume | unrealised pnl | realised pnl |
+      | auxiliary1 | -10    | 0              | 0            |
+      | auxiliary2 | 10     | 0              | 0            |
+   # setup trader3 position and close it out
+    When the parties place the following orders:
+      | party   | market id | side | volume| price | resulting trades | type       | tif     | reference      |
+      | trader2 | ETH/DEC19 | sell | 10    | 100   | 1                | TYPE_LIMIT | TIF_GTC | sell-provider-1|
```

Comment:
Wouldn't it be cleaner to use one of the auxiliary traders to move the market here? I struggle to understand if trader2 really get's forcibly closed out or if he just trades -10 here and then his +10 order trades with networks effectively closing him out (not "forcibly" though if that makes sense). Might also be worth adding a check for the resulting trades to demonstrate that `trader3` acquires a position of +10 as a result of this order.

Reply:
replaced trader2 with auxiliary2